ibis/backends/postgres/datatypes.py

```diff
@@ -1,46 +1,12 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING
-
 import sqlalchemy as sa
 import sqlalchemy.dialects.postgresql as psql
 import sqlalchemy.types as sat
 
 import ibis.expr.datatypes as dt
 from ibis.backends.base.sql.alchemy.datatypes import AlchemyType
-from ibis.common.collections import FrozenDict
-from ibis.formats.parser import TypeParser
-
-if TYPE_CHECKING:
-    from collections.abc import Mapping
-
-
-class PostgresTypeParser(TypeParser):
```

Comment:
Moved to `base/sql/glot/datatypes.py` for centralization, probably I should move it back.

Reply:
Put up a PR: https://github.com/tobymao/sqlglot/pull/2081